src/cljs/rems/administration/create_form.cljs

```diff
@@ -177,13 +177,14 @@
  ::send-form
  (fn [{:keys [db]} [_]]
    (let [edit? (::edit-form? db)
-         form-errors (validate-form-template (get-in db [::form :data]) (:languages db))
+         form-preview-data (build-request (get-in db [::form :data]) (:languages db))
+         form-errors (validate-form-template form-preview-data (:languages db))
```

Comment:
huh, that's all the fix really needs? cool

Reply:
well, it works (on my machine haha) and acts according to the spec